cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -52,7 +52,7 @@ macro (build_addon target prefix libs)
     # Read used headers from addon, needed to identitfy used kodi addon interface headers
     if(${prefix}_HEADERS)
       # Add the used header files defined with CMakeLists.txt from addon itself
-      if(${prefix}_HEADERS MATCHES ${PROJECT_SOURCE_DIR})
+      if(${prefix}_HEADERS STREQUAL ${PROJECT_SOURCE_DIR})
```

Comment:
@AlwinEsch was your intention to use  `MATCHES` as exact string match or some kind of StartsWith, which cmake doesn't have?

Reply:
No, was here during test the way who has worked :).
If "STREQUAL" can do the same it is also OK. Has not thought about circumstance with "+" in path.